code/game/gamemodes/clock_cult/clock_structures/heralds_beacon.dm

```diff
@@ -104,6 +104,7 @@
 		if(is_servant_of_ratvar(H))
 			to_chat(H, "<span class='bold alloy'>The beacon's power warps your body into a clockwork form! You are now immune to many hazards, and your body is more robust against damage!</span>")
 			H.set_species(/datum/species/golem/clockwork/no_scrap)
+			H.add_overlay(mutable_appearance('icons/effects/genetics.dmi', "servitude", -MUTATIONS_LAYER))
```

Comment:
I don't know too much about mob overlays, but I feel like this won't stay when icons are regenerated

Reply:
This probably means you can game regular clock cult as it is by using mutation tox from xenobio or something to cut your overlays

Reply:
I recall implementing some sort of failsafe for this, but not where or how.

Reply:
they should use the priority overlay system, because that's what it's for